data/textile/export.py

```diff
@@ -1,151 +1,208 @@
 #!/usr/bin/env python
-# coding: utf-8
 
-"""Export des matières et procédés du textile"""
+"""Materials and processes export for textile"""
 
-import json
+import os
+import sys
+from os.path import dirname
 
-import bw2calc
-from bw2data.project import projects
+import bw2data
+import pandas as pd
+from common import (
+    fix_unit,
+    order_json,
+    remove_detailed_impacts,
+    with_aggregated_impacts,
+    with_corrected_impacts,
+)
 from common.export import (
+    IMPACTS_JSON,
     cached_search,
+    check_ids,
+    compare_impacts,
+    compute_impacts,
     display_changes,
-    with_corrected_impacts,
-    with_subimpacts,
+    export_json,
+    load_json,
+    plot_impacts,
 )
-from common.impacts import impacts as impacts_definition
+from common.impacts import impacts as impacts_py
+from frozendict import frozendict
+
+BW_DATABASES = bw2data.databases
+PROJECT_ROOT_DIR = dirname(dirname(dirname(__file__)))
+ECOBALYSE_DATA_DIR = os.environ.get("ECOBALYSE_DATA_DIR")
+if not ECOBALYSE_DATA_DIR:
+    print(
+        "\n🚨 ERROR: For the export to work properly, you need to specify ECOBALYSE_DATA_DIR env variable. It needs to point to the https://github.com/MTES-MCT/ecobalyse-private/ repository. Please, edit your .env file accordingly."
+    )
+    sys.exit(1)
+
+# Configuration
+DEFAULT_DB = "Ecoinvent 3.9.1"
+ACTIVITIES_FILE = f"{PROJECT_ROOT_DIR}/data/textile/activities.json"
+COMPARED_IMPACTS_FILE = f"{PROJECT_ROOT_DIR}/data/textile/compared_impacts.csv"
+IMPACTS_FILE = f"{PROJECT_ROOT_DIR}/public/data/impacts.json"
+MATERIALS_FILE = f"{PROJECT_ROOT_DIR}/public/data/textile/materials.json"
+PROCESSES_IMPACTS = f"{ECOBALYSE_DATA_DIR}/data/textile/processes_impacts.json"
+PROCESSES_AGGREGATED = f"{PROJECT_ROOT_DIR}/public/data/textile/processes.json"
+GRAPH_FOLDER = f"{PROJECT_ROOT_DIR}/data/textile/impact_comparison"
+
+
+def create_material_list(activities_tuple):
+    print("Creating material list...")
+    return tuple(
+        [
+            to_material(activity)
+            for activity in list(activities_tuple)
+            if activity["category"] == "material"
+        ]
+    )
+
+
+def to_material(activity):
+    return {
+        "id": activity["material_id"],
+        "materialProcessUuid": activity["uuid"],
+        "recycledProcessUuid": activity.get("recycledProcessUuid"),
+        "recycledFrom": activity.get("recycledFrom"),
+        "name": activity["shortName"],
+        "shortName": activity["shortName"],
+        "origin": activity["origin"],
+        "primary": activity.get("primary"),
+        "geographicOrigin": activity["geographicOrigin"],
+        "defaultCountry": activity["defaultCountry"],
+        "priority": activity.get("priority"),
```

Comment:
This field is never used by anything in the entire codebase it seems, and should be therefore removed entirely.

Reply:
Thx! You mean the `priority` field?
Btw I'll do another PR to do such changes

Reply:
yes, the priority field